src/main/java/com/rocana/lucene/codec/v1/RocanaBlockTreeTermsReader.java

```diff
@@ -44,7 +45,28 @@
 import org.apache.lucene.util.fst.ByteSequenceOutputs;
 import org.apache.lucene.util.fst.Outputs;
 
-/** A block-based terms index and dictionary that assigns
+/**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.RocanaBlockTreeTermsReader}.
+ *
+ * Why we forked:
+ *   - We can't extend because it has final fields that must be set in the constructor,
+ *     and the constructor is the code we need to modify.
+ *
+ * What changed in the fork?
+ *   - Commented out the constructor's call to {@link CodecUtil#checksumEntireFile(IndexInput)}
+ *     for performance.
+ *   - Use the other forked classes, like {@link RocanaFieldReader}.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * To see a full diff of changes: compare this version to this file's initial commit (which
+ * is the original file from Lucene).
+ *
+ * See ROCANA-8229.
```

Comment:
I'll remove it from all the forked files. For the benefit of the Rocana engineer (and let's be honest, there's a 90% chance this code's only ever be used by Rocana engineers) I'll keep the reference in the main RocanaSearchCodecV1 javadocs, and say what we have in the README.md:

> If you have access to Rocana's JIRA, see ROCANA-8229.